### PR TITLE
Platform upgrade - closes #35 and closes #31

### LINKED ISSUE #31: Add local NTP server and enable NTP configuration during install
Every node and service VM should use a fallback NTP server in case the primary NTP is down. The NTP server should be configurable during install.

Should erigonesd depend on the ntp service?

- Adjust triggers in monitoring.

- Update documentation (user guide).

### LINKED ISSUE #35: Update platform image
Remove error message: "platfom/i686/kernel/unix No such file or directoyr"

### PR DESCRIPTION
**Pull request by dn0**
closes #35 and closes #31

- Improved cfgdb availability check
- Added /esdc/settings/dc tree into cfgdb
- Backported changes from upstream - related to platform upgrade - #35
- Removed old prompt-configs
- Changed `Headnode` to `Compute node` - we want to remove the headnode concept - related to erigones/esdc-docs#13
- Added support for admin_vlan_id on head node
- The admin interface with VLAN will be called admin0
- Reboot stderr goes to /dev/null to hide the bootadm update-archive message
- Added admin_vlan_id checking and writing admin_vlan_id into config file
- Added creation of `zones/backups/manifests` - erigones/esdc-ce#155
- Enabled nicsdown and moved nicsdown before install verification
- Nicer messages during HDD install
- Updated to new platform version - 20170513T080150Z from upstream/release-20170511

### CODE REVIEW

File: ansible/templates/usb/scripts/computenode.sh.j2 [hunk 1]
```diff
@@ -137,10 +137,11 @@ function wait_for_cfgdb() {
 	set +e
 	while true; do
 		printf_log "\r%-64s" "Waiting for cfgdb availability... "
-		/usr/sbin/ping -s -n -I 1 ${CFGDB_IP} 1492 10 > /dev/null 2>&1
-		if [ $? -eq 0 ]; then
-			printf_log "done\n"
-			break
+		if ping -n ${CFGDB_IP} > /dev/null 2>&1; then
+			if echo "ruok" | nc -n -w 10 10.0.60.106 2181 2> /dev/null | grep imok >/dev/null 2>&1; then
```
Comment (YanChii) [inline ansible/templates/usb/scripts/computenode.sh.j2]: LOL :)

File: ansible/templates/usb/boot/menu.lst.j2 [hunk 2]
```diff
@@ -6,7 +6,6 @@ terminal composite
 variable os_console vga
 splashimage=/boot/splash.xpm.gz
 
-{% if usb_type == "cn" %}
 title Compute Node 64-bit
```
Comment (YanChii) [inline ansible/templates/usb/boot/menu.lst.j2]: Are we sure we don't want to distinguish between first node (full) USB key install and the other USB keys? At least by some comment.

File: ansible/templates/usb/scripts/computenode.sh.j2 [hunk 3]
```diff
@@ -229,8 +230,8 @@ function reload_smf_manifests() {
 	printf_log "\r%-64s" "Reloading SMF manifests... "
 	printf_log "%4s\n" "done"
 	# This will make the output to look nicer/aligned
-	/usr/sbin/svcadm disable -s svc:/system/manifest-import:default
-	/usr/sbin/svcadm enable -s svc:/system/manifest-import:default
+	svcadm disable -s svc:/system/manifest-import:default
+	svcadm enable -s svc:/system/manifest-import:default
```
Comment (YanChii) [inline ansible/templates/usb/scripts/computenode.sh.j2:234]: Is it good idea to use relative paths?

Comment (dn0) [reply ansible/templates/usb/scripts/computenode.sh.j2:234]: It depends... In this case, we have defined a PATH, so it is better to use relative paths.

**Review by YanChii — APPROVED** [review]

**Pull request merged**